content/pt-br/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,528 @@
+| Prefixo | `/aaa/bbb`                       | `/aaa/bbb/ccc`                | Sim, corresponde ao subcaminho     |
+| Prefixo | `/aaa/bbb`                       | `/aaa/bbbxyz`                 | Não, não corresponde ao prefixo da string   |
+| Prefixo | `/`, `/aaa`                      | `/aaa/ccc`                    | Sim, corresponde ao prefixo `/aaa` |
+| Prefixo | `/`, `/aaa`, `/aaa/bbb`          | `/aaa/bbb`                    | Sim, corresponde ao prefixo `/aaa/bbb` |
+| Prefixo | `/`, `/aaa`, `/aaa/bbb`          | `/ccc`                        | Sim, corresponde ao prefixo `/`    |
+| Prefixo | `/aaa`                           | `/ccc`                        | Não, usa o backend padrão          |
+| Misto   | `/foo` (Prefixo), `/foo` (Exato) | `/foo`                        | Sim, prefere o exacto              |
+
+#### Várias correspondências
+
+Em alguns casos, vários caminhos dentro de uma entrada corresponderão a uma solicitação. 
+Nesses casos, a precedência será dada primeiro ao caminho correspondente mais longo. 
+Se dois caminhos ainda estiverem iguais, a precedência será dada aos caminhos com um tipo de caminho exato sobre o tipo de caminho de prefixo.
+
+## Hostname curingas
+
+Os hosts podem ter correspondências precisas (por exemplo, “foo.bar.com”) ou um curinga (por exemplo, “`*.foo.com`”). 
+Correspondências precisas exigem que o cabeçalho do `host` HTTP corresponda ao campo `host`. 
+As correspondências curinga exigem que o cabeçalho do `host` HTTP seja igual ao sufixo da regra curinga.
+
+
+| Host        | Host header       | Corresponde?                                            |
+| ----------- |-------------------| --------------------------------------------------|
+| `*.foo.com` | `bar.foo.com`     | Correspondências baseadas no sufixo compartilhado                    |
+| `*.foo.com` | `baz.bar.foo.com` | Sem correspondência, o curinga cobre apenas um único rótulo DNS |
+| `*.foo.com` | `foo.com`         | Sem correspondência, o curinga cobre apenas um único rótulo DNS |
+
+{{< codenew file="service/networking/ingress-wildcard-host.yaml" >}}
+
+## Classe Ingress
+
+As entradas podem ser implementadas por diferentes controladores, muitas vezes com diferentes configurações. 
+Cada Ingress deve especificar uma classe, uma referência a um recurso IngressClass que contém uma configuração adicional, incluindo o nome do controlador que deve implementar a classe.
+
+{{< codenew file="service/networking/external-lb.yaml" >}}
+
+O campo `.spec.parameters` de uma classe Ingress permite que você faça referência a outro recurso que fornece a configuração relacionada a essa classe Ingress.
+
+O tipo específico de parâmetros a serem usados depende do controlador de entrada que você especificar no campo `.spec.controller` da classe Ingress.
+
+### Escopo da classe Ingress
+
+Dependendo do seu controlador de entrada, você pode ser capaz de usar parâmetros que você define em todo o cluster ou apenas para um namespace.
```

Comment:
Although I agree with the use of active voice, I think the sentence "você pode ser capaz de..." is sounding a little weird. Perhaps using passive voice would be more adequate here?

Reply:
I rewrote the sentence. Please, take a look and see if it's better.